packages/generator-ds/src/sv-component/templates/Component.svelte.test.ts.ejs

```diff
@@ -1,28 +1,80 @@
 /* <%= generatorPackageName %> <%= generatorPackageVersion %> */
 
+import type { Locator } from "@vitest/browser/context";
 import { createRawSnippet } from "svelte";
 import { describe, expect, it } from "vitest";
 import { render } from "vitest-browser-svelte";
+import type { RenderResult } from "vitest-browser-svelte";
 import Component from "./<%= componentName %>.svelte";
+import type { <%= componentName %>Props } from "./types.js";
```

Comment:
can we try to remove this import from both test templates, instead infer the props from the imported generic `Component` to avoid maintain the type props name in multiple files in a case a component gets renamed during development for example.